developers.md

```diff
@@ -1,28 +1,82 @@
 # rules_pkg - Information for Developers
```

Comment:
It might be worth linking to this document from an [issue template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository).  I can't seem to configure it myself -- perhaps you can?

Consider also capitalizing this file name to make it stand out more.

Reply:
Maybe. Let's discuss that offline. In any case, that would nto change the wording here.